pkg/materialize/format_specs.go

```diff
@@ -21,7 +21,7 @@ type FormatSpecStruct struct {
 	Avro     *AvroFormatSpec
 	Protobuf *ProtobufFormatSpec
 	Csv      *CsvFormatSpec
-	Json     bool
+	Bytes    bool
```

Comment:
Generic question unrelated to the PR, but is there a reason for us to not name this type `SourceFormatSpecStruct`, if we then use `SinkFormatSpecStruct` for sinks? @dehume

Reply:
For consistency, we probably should change it.

Reply:
Good point! Just updated this accordingly!